addon/components/one-way-input.js

```diff
@@ -86,12 +86,16 @@ const OneWayInputComponent = Component.extend({
   },
 
   init() {
-    this._super(...arguments);
-
     let value = get(this, 'paramValue') || get(this, 'value');
     set(this, 'value', value);
 
     this._sanitizedValue = get(this, 'value') || get(this, 'checked');
+    this._super(...arguments);
+  },
+
+  didReceiveAttrs() {
+    this._super(...arguments);
+    this._processNewValue.call(this, undefined, get(this, get(this, 'appropriateAttr')));
```

Comment:
Seems odd to use `.call` when it has the same `this` with and without.

I would think this would be the same thing:

```js
this._processNewValue(undefined, get(this, get(this, 'appropriateAttr')));
```